ios/browser/api/certificate/models/brave_certificate_signature.mm

```diff
@@ -24,10 +24,10 @@ - (instancetype)initWithCertificate:(const net::ParsedCertificate*)certificate {
 
     _digest = base::SysUTF8ToNSString(
         certificate::x509_utils::SignatureAlgorithmDigestToName(
-            certificate->signature_algorithm()));
+            *certificate->signature_algorithm()));
```

Comment:
shouldn't we be checking has_value here if these are optional?

Reply:
@kylehickinson has raised this above in another comment. I'm waiting for a reply there.

Reply:
@mkarolin Unfortunately that fix introduces a potential crash as well because those properties are marked `nonnull` by default. Swift will import them as such and if there's a nullptr under it–which would happen if `has_value` returns false–it will crash.  We have 2 options here:

- Make the `digest` and `algorithm` properties nullable (i.e in `brave_certificate_signature.h`, doing `@property(nonatomic, readonly, nullable) ...` on the two properties)
- Give the `digest` and `algorithm` default values such as empty strings.

I'd prefer the former, and we can deal with the optionals in `brave-ios`.

Reply:
@kylehickinson wouldn't 
<pre>
    _algorithm = [[NSString alloc] init];
    _digest = [[NSString alloc] init];
</pre>
have initialize them to empty strings?

Reply:
@mkarolin yeah that would do it, sorry I'm blind! Carry on... 🙂